src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/AbstractPlatformProxy.cs

```diff
@@ -199,5 +199,10 @@ public virtual IPoPCryptoProvider GetDefaultPoPCryptoProvider()
         {
             throw new NotImplementedException();
         }
+
+        public virtual IDeviceAuthManager CreateDeviceAuthManager()
+        {
+            throw new NotImplementedException();
```

Comment:
What will happen on platforms where you do not define a platform proxy, like .net core, .net standard or xamarin.mac?

Reply:
I suggest that you create a NotSupportedDeviceAuthManager for these platforms for now.

Reply:
Yes